WordPress/src/test/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/LatestPostSummaryMapperTest.kt

```diff
@@ -86,7 +97,18 @@ class LatestPostSummaryMapperTest {
     @Test
     fun `builds message with engagement`() {
         val viewCount = 10
-        val model = InsightsLatestPostModel(siteId, postTitle, postURL, date, postId, viewCount, 0, 0, listOf())
+        val model = InsightsLatestPostModel(
+                siteId,
+                postTitle,
+                postURL,
+                date,
+                postId,
+                viewCount,
+                0,
+                0,
+                listOf(),
+                ""
```

Comment:
Nitpick: Although the value passed is empty, I think we could add a variable with the default value like `val featuredImageURL = ""` and use it here for future purposes. WDYT. 🤔

Reply:
Yes, you're right! Done in 51201b97d5fda22d3d7cd2519d8b10b71ba744ab

Reply:
@irfano Feel free to merge once the CI is completed successfully. 👍🏼 🙌🏼